installer/pyinstaller/hook-samcli.py

```diff
@@ -13,6 +13,9 @@
     )[0]
     + hooks.collect_all("jschema_to_python", include_py_files=False)[0]
     + hooks.collect_all("cfnlint", include_py_files=True)[0]
+    # cookiecutter 2.2.3 added the version in a file called VERSION.txt which needs to be added
+    # for sam init to run cookiecutter using pyinstaller https://github.com/cookiecutter/cookiecutter/releases/tag/2.2.3
+    + hooks.collect_all("cookiecutter", include_py_files=False)[0]
```

Comment:
Can we do sanity check for `sam init` command with the pyinstaller artifacts generated by this PR?

Reply:
Synced offline about this, and it is already verified in pyinstaller ci/cd job. It is also verified manually